pytest.ini

```diff
@@ -3,3 +3,6 @@
 norecursedirs = dataset
 python_classes = Pytest
 python_functions = pytest_
+filterwarnings =
```

Comment:
@streeve 
do we really want to ignore warnings about deprecated capabilities? Aww... This is just for the unit tests. Never mind

Reply:
Right, I just don't think we need it in the pytest output. We should add a flake/lint step though...